packages/console/src/components/TabNav/TabNavLink.module.scss

```diff
@@ -1,18 +1,22 @@
 @use '@/scss/underscore' as _;
 
 .link {
-  font: var(--font-body-medium);
+  font: var(--font-subhead-2);
   margin-right: _.unit(6);
   padding-bottom: _.unit(1);
 
   a {
-    color: unset;
+    color: var(--color-component-caption);
     text-decoration: none;
   }
 }
 
 .selected {
-  color: var(--color-primary);
-  border-bottom: 1px solid var(--color-primary);
+  color: var(--color-primary-40);
```

Comment:
`--color-primary` is `--color-primary-40`, we don't need to change it here.
And actually, try avoid using `--color-blahblah-10/20/30...` directly in our code, use those color tokens defined in theme mixin instead.

Reply:
updated, PTAL